http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/db/FollowupComponent.scala

```diff
@@ -0,0 +1,44 @@
+package org.broadinstitute.dsde.workbench.leonardo
+package db
+
+import org.broadinstitute.dsde.workbench.leonardo.monitor.ClusterFollowupDetails
+
+import scala.concurrent.ExecutionContext
+import LeoProfile.api._
+
+case class FollowupRecord(clusterId: Long, status: String, masterMachineType: Option[String])
+
+class FollowupTable(tag: Tag) extends Table[FollowupRecord](tag, "CLUSTER_FOLLOWUP") {
+  def clusterId = column[Long]("clusterId", O.PrimaryKey)
+  def status = column[String]("status", O.Length(254))
```

Comment:
can we do `column[ClusterStatus]` here, and then in `mappedColumnImplicits`, and something similar to 
```
    implicit val gsPathMappedColumnType: BaseColumnType[GcsPath] =
      MappedColumnType
        .base[GcsPath, String](_.toUri, s => parseGcsPath(s).fold(e => throw new Exception(e.toString()), identity))
```

Reply:
I did this ` MappedColumnType .base[ClusterStatus, String](_.entryName, s => ClusterStatus.withName(s))`. Not sure if I need to make a helper function to lift it to an either, or if the fact withName throws an exception is sufficient?

Reply:
yea, here `ClusterStatus.withName(s))` would just throw an exception